composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle": "~2.3",
+        "symfony/framework-bundle": "~2.7.5",
```

Comment:
Yes, it's just a mistake, because I had to add some packages to the composer.json in local environment in order to run unit tests.